releases/release-8.3.0.md

```diff
@@ -0,0 +1,410 @@
+---
+title: TiDB 8.3.0 Release Notes
+summary: 了解 TiDB 8.3.0 版本的新功能、兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 8.3.0 Release Notes
+
+发版日期：2024 年 x 月 x 日
+
+TiDB 版本：8.3.0
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v8.3/quick-start-with-tidb) | [下载离线包](https://cn.pingcap.com/product-community/?version=v8.3.0-DMR#version-list)
+
+在 8.3.0 版本中，你可以获得以下关键特性：
+
+<table>
+<thead>
+  <tr>
+    <th>分类</th>
+    <th>功能/增强</th>
+    <th>描述</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="4">可扩展性和性能</td>
+    <td> <a href="https://docs.pingcap.com/zh/tidb/v8.3/partitioned-table#全局索引">分区表全局索引（实验特性）</a></td> **tw@hfxsd** <!--1531-->
+    <td>全局索引能够有效提升对非分区键的检索效率，同时也解除了分区键一定要包含唯一键 (Unique Key) 的限制，扩展了 TiDB 分区表的使用场景，也能够避免数据迁移可能遇到的部分应用改造工作。</td>
+  </tr>
+  <tr>
+    <td>Projection 下推成为正式功能</td>**tw@Oreoxmt** <!--1872-->
+    <td> Projection 下推能够尽可能将负载分散到存储节点，也可以减少节点间的数据传输，这能够降低一部分 SQL 的执行时间，提升数据库整体性能。</td>
+  </tr>
+  <tr>
+    <td>统计信息收集忽略不必要的列</td>**tw@lilin90** <!--1753-->
+    <td>在保证优化器能够获取到必要信息的前提下，加快了统计信息收集的速度，提升统计信息的时效性，进而保证最优的执行计划的选择，提升集群性能。同时也降低的系统开销，改善资源利用率。</td>
+  </tr>
+  <tr>
+    <td>读写性能的细粒度优化</td>**tw@qiancai** <!--1893-->
+    <td>通过优化 KV 请求的策略，增加获取 TSO 的模式等多重手段，进一步提升 TiDB 的读写性能，降低业务的执行时间，改善延迟。</td>
+  </tr>
+  <tr>
+    <td rowspan="1">稳定性与高可用</td>
+    <td>TiProxy 管理虚拟 IP</td>**tw@Oreoxmt** <!--1887-->
+    <td>在 TiProxy 内实现了对虚拟 IP 的管理，支持自动的虚拟 IP 切换，而不依赖外部平台或工具。这能够简化 TiProxy 的部署形式，降低了数据库接入层的复杂度。</td>
+  </tr>
+</tbody>
+</table>
+
+## 功能详情
+
+### 可扩展性
+
+* 功能标题 [#issue号](链接) @[贡献者 GitHub ID](链接) **tw@xxx** <!--1234-->
+
+    功能描述（需要包含这个功能是什么、在什么场景下对用户有什么价值、怎么用）
+
+    更多信息，请参考[用户文档](链接)。
+
+### 性能
+
+* 功能标题 [#issue号](链接) @[贡献者 GitHub ID](链接) **tw@xxx** <!--1234-->
+
+    功能描述（需要包含这个功能是什么、在什么场景下对用户有什么价值、怎么用）
+
+    更多信息，请参考[用户文档](链接)。
+
+* Projection 下推功能正式发布 [#51876](https://github.com/pingcap/tidb/issues/51876) @[yibin87](https://github.com/yibin87) **tw@Oreoxmt** <!--1872-->
+
+    将 `Projection` 算子下推到存储引擎可以减少计算引擎和存储引擎之间的传输的数据量，特别是对 [JSON 查询类函数](/functions-and-operators/json-functions/json-functions-search.md)或 [JSON 值属性类函数](/functions-and-operators/json-functions/json-functions-return.md)，效果更加显著。在 v8.3.0 版本中，TiDB 正式发布 Projection 下推功能。在该功能启用时，优化器会自动将符合条件的 JSON 查询类函数、JSON 值属性类函数等下推至存储引擎。
+    
+    变量 [tidb_opt_projection_push_down](/system-variables.md#tidb_opt_projection_push_down-从-v610-版本开始引入) 控制是否启用 Projection 下推功能。该变量的默认值将调整为 `ON`，表示默认启用该功能。
+
+    更多信息，请参考[用户文档](/system-variables.md#tidb_opt_projection_push_down-从-v610-版本开始引入)。
+    
+* 优化 KV 请求的批处理策略 [#xxx](https://github.com/pingcap/tidb/issues/xxx) @[zyguan](https://github.com/zyguan) **tw@Oreoxmt** <!--1897-->
```

Comment:
```suggestion
* 优化 KV 请求的批处理策略 [#](https://github.com/pingcap/tidb/issues/) @[zyguan](https://github.com/zyguan) **tw@Oreoxmt** <!--1897-->
```
@zyguan Please provide the GitHub issue link

Reply:
https://github.com/pingcap/tidb/issues/55206